core/main/src/Options/Classes/Options.ts

```diff
@@ -102,12 +105,12 @@ export class Options implements IOptions, IOptionLoader<IOptions> {
         if (data.pauseOnBlur !== undefined) {
             this.pauseOnBlur = data.pauseOnBlur;
         }
-
         this.background.load(data.background);
         this.particles.load(data.particles);
         this.infection.load(data.infection);
         this.interactivity.load(data.interactivity);
         this.backgroundMask.load(data.backgroundMask);
+        this.themes.load(data.themes);
```

Comment:
This needs a for over `data.themes` if not `undefined`

Reply:
sth like
 if(data.themes !== undefined){
            for(const theme in data.themes){
                this.themes.push(theme.load(themes))
            }
        }